cypress/integration/aaaaaaaaaaaaaaaaaaaaaaaa/aaaaaaaaaaaa.js

```diff
@@ -43,41 +51,57 @@ describe('Read fixture and prepare test data', function() {
     cy.getCurrentWindowRecordId().then(id => (priceListID = id));
   });
 
-  // it('Create Product1 and Category', function() {
-  //   const productCategoryJson = fixture['ProductCategory'];
-  //   productCategoryJson.name = productCategoryJson + appendDate();
-  //   categoryName = productCategoryJson.name;
-  //
-  //
-  //   Builder2.createProductEntities(productCategoryJson, priceListName, productName1, productName1, productType);
-  // });
+  it('Create Product1 and Category', function() {
+    const productCategoryJson = fixture['ProductCategory'];
+    productCategoryJson.name = productCategoryJson.name + appendDate();
+    categoryName = productCategoryJson.name;
+
+    const productJson = fixture['Products'][0];
+    productJson.name = productJson.name + appendDate();
```

Comment:
Can't we put appendDate() directly int he fixture?

Reply:
I don't think we can. If we add a specific date, then all future runs have the same date, so our test data is no longer unique.

If what you ask is to make a name in the fixture like "Product1_`+ appendDate()`" then i guess we could do that but using [eval](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) is a can of worms i will not touch even wearing 2 hazard suits at the same time ☣️ and will not approve of code using that.